packages/tslint/index.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * Tslint webpack block.
+ *
+ * @see https://github.com/wbuchwalter/tslint-loader
+ */
+
+module.exports = tslint
+
+/**
+ * @param {object}   [options]                  See https://github.com/wbuchwalter/tslint-loader#usage
+ * @return {Function}
+ */
+function tslint (options) {
+  options = options || {}
+
+  const loader = context => ({
+    test: context.fileType('application/x-typescript'),
+    loaders: [ 'tslint-loader' ]
+  })
+
+  const module = context => (context.webpackVersion.major === 1 ? {
+    preLoaders: [ loader(context) ]
+  } : {
+    loaders: [ Object.assign({}, loader(context), {
```

Comment:
Might be easier readable as `loaders: [ loader(context, { enforce: 'pre' }) ]`. What do you think?